tests.py

```diff
@@ -11,3 +18,24 @@ def test_render():
     c = Context()
     t = Template('{% extends "admin:admin/base.html" %}')
     t.render(c)
+
+
+overrides = {}
```

Comment:
What do you think, should this better be named `cached_templates`? Maybe this would be more self-explanatory.

Reply:
Have gone for `cached_template_settings`